internal/services/nss/nss_test.go

```diff
@@ -73,7 +72,7 @@ func TestGetPasswdByName(t *testing.T) {
 			client := newNSSClient(t, tc.sourceDB, false)
 
 			got, err := client.GetPasswdByName(context.Background(), &authd.GetPasswdByNameRequest{Name: tc.username, ShouldPreCheck: tc.shouldPreCheck})
-			requireExpectedResult(t, "GetPasswdByName", got, err, tc.wantErr, tc.wantErrNotExists)
```

Comment:
is `tc.wantErrNotExists` still on the test struct? Is it still used? Don’t we loose information that we can retest by another mean? (IIRC, we trasnform the error not exist on the client nss lib?)